packages/avado/dappnode_package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hopr.avado.dnp.dappnode.eth",
-  "version": "0.100.0",
-  "description": "The HOPR protocol ensures everyone has control of their privacy, data, and identity. By running a HOPR Node, you can obtain HOPR tokens by relaying data and connect to the HOPR Network.",
+  "version": "0.0.0",
```

Comment:
Why is this better or more correct?

Reply:
It's just a placeholder and _always_ gets changed by the build script.